health_vara_view.xml

```diff
@@ -3,48 +3,16 @@
 contains the full copyright notices and license terms. -->
 <tryton>
     <data>
-        <!-- Mammography Party Views -->
-        <record model="ir.ui.view" id="mammography_party_tree">
-            <field name="model">party.party</field>
-            <field name="type">tree</field>
-            <field name="name">party_tree</field>
-        </record>
-        <record model="ir.ui.view" id="mammography_party_form">
-            <field name="model">party.party</field>
-            <field name="type">form</field>
-            <field name="name">party_form</field>
-        </record>
-
-        <!-- Mammography Patient Views -->
-        <record model="ir.ui.view" id="mammography_patient_tree">
-            <field name="model">gnuhealth.patient</field>
-            <field name="type">tree</field>
-            <field name="name">mammography_patient_tree</field>
-        </record>
         <record model="ir.ui.view" id="mammography_patient_form">
             <field name="model">gnuhealth.patient</field>
-            <field name="type">form</field>
+            <field name="inherit" ref="health.gnuhealth_patient_view_form"/>
```

Comment:
I wanted to completely replace the definition of these by id, but it seemed not to have an affect. Would rather do it that way since the 'inheriting' xml just replaces everything.